### PR TITLE
Add funding account module

### PR DESCRIPTION
**Pull request by zkhin**

### CODE REVIEW

File: src/components/funding-account/funding-account.service.ts [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+    });
+  }
+
+  async delete(id: string, session: ISession): Promise<void> {
+    const fundingAccount = await this.readOne(id, session);
+    try {
+      await this.db.deleteNode({
+        session,
+        object: fundingAccount,
+        aclEditProp: 'canDeleteOwnUser',
```
Comment (michaelmarshall) [inline src/components/funding-account/funding-account.service.ts:182]: we are in the middle of refactoring how we delete nodes. this function and the test can be 'non-implemented' for now.

File: test/funding-account.e2e-spec.ts [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        },
+      }
+    );
+    const updated = result.updateFundingAccount.fundingAccount;
+    expect(updated).toBeTruthy();
+    expect(updated.name.value).toBe(newName);
+  });
+
+  // Delete FundingAccount
+  it('delete funding account', async () => {
```
Comment (michaelmarshall) [inline test/funding-account.e2e-spec.ts:86]: best to skip this until our delete refactor is done.

File: src/components/funding-account/funding-account.service.ts (hunk 1 above)
Comment (michaelmarshall) [reply src/components/funding-account/funding-account.service.ts:182]: fyi, we won't be using `acl` concepts at all.

Comment (zkhin) [reply src/components/funding-account/funding-account.service.ts:182]: Okay

**Review by CarsonF — APPROVED** [review]

**Pull request merged**